packages/backend/src/lib/balancerPolygonClient.ts

```diff
@@ -0,0 +1,10 @@
+import { GraphQLClient } from 'graphql-request';
+
+import { getSdk } from './autogen/balancerpolygon-sdk';
+
+const balancerPolygonGraphqlURL =
+  'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-polygon-v2';
```

Comment:
Should this be in constants?

Reply:
I left this here because presumably it would only be used here

Reply:
> I left this here because presumably it would only be used here

For me, the constants file is the place I'd go if wanting to customize the behavior of the application. If I were looking to find where that value is defined, I'd look in constants.